lerna.json

```diff
@@ -4,6 +4,7 @@
   ],
   "npmClient": "yarn",
   "useWorkspaces": true,
+  "useNx": true,
```

Comment:
Major change in Lerna 6 is the new task runner `nx`. We can revert to a legacy behavior by setting this to false.